servant-client/test/Servant/ClientSpec.hs

```diff
@@ -155,6 +158,7 @@ server = serve api (
        return alice
   :<|> return NoContent
   :<|> (\ name -> return $ Person name 0)
+  :<|> (\ (name : _) -> return $ Person name 0)
```

Comment:
Good point. Changed as per your suggestion (although I've used parens rather than `$` to be consistent w/ existing code).